pkg/v1/platform.go

```diff
@@ -23,3 +23,23 @@ type Platform struct {
 	Variant      string   `json:"variant,omitempty"`
 	Features     []string `json:"features,omitempty"`
 }
+
+// Equal return if another platform is equivalent to this one. It must have all of the fields be the same
+func (p Platform) Equal(o Platform) bool {
```

Comment:
I _think_ conventionally you'd use `Equals` when this is a method, e.g. `foo.Equals(bar)`. I do see a lot of `Equal` for when this is a function, e.g. `reflect.DeepEqual` and `bytes.Equal`.

Reply:
I don't mind changing it. Hang tight.

Reply:
Updated.

Reply:
godoc will be mad about the comment 🙃